remote_syslog.go

```diff
@@ -53,6 +54,11 @@ func tailOne(file string, excludePatterns []*regexp.Regexp, logger *syslog.Logge
 // Tails files speficied in the globs and re-evaluates the globs
 // at the specified interval
 func tailFiles(globs []string, excludedFiles []*regexp.Regexp, excludePatterns []*regexp.Regexp, interval RefreshInterval, logger *syslog.Logger, severity syslog.Priority, facility syslog.Priority, poll bool) {
+	if len(globs) == 0 {
```

Comment:
It prevents running of remote syslog if no files specified. As r_s2 does not support `-HUP` signal to reload configuration, it has no sense to keep it running with empty files list to watch, imho.